admin-dev/themes/new-theme/js/pages/order/create/product-renderer.js

```diff
@@ -73,6 +73,12 @@ export default class ProductRenderer {
         $template.find(createOrderMap.listedProductQtyInput).data('attribute-id', product.attributeId);
         $template.find(createOrderMap.listedProductQtyInput).data('customization-id', customizationId);
         $template.find(createOrderMap.listedProductQtyInput).data('prev-qty', product.quantity);
+        this.renderStock(
+          $template.find(createOrderMap.listedProductQtyStock),
+          $template.find(createOrderMap.listedProductQtyInput),
+          product.availableStock,
+          product.availableOutOfStock || (product.availableStock <= 0)
```

Comment:
This check is weird, I know it's not new from your PR but how does it make sense?
If the stock is negative or zero then we consider it infinite?

Reply:
Yes